### PR TITLE
docs: improve configuration development documentation

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Rewrite sentence
- Use one sentence per line style
- Ask question in file about environment

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

File: docs/development/configuration.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ## Configuration Methods
 
-Configuration is supported via any or all of the below:
+The Renovate bot can be configured via any of these methods:
 
 - Configuration file
-- Environment
+- Environment (variable? file? what is this about?)
```
Comment (HonkingGoose) [inline docs/development/configuration.md:8]: Can somebody answer what this is about?

File: docs/development/configuration.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 ## Configuration Methods
 
-Configuration is supported via any or all of the below:
+The Renovate bot can be configured via any of these methods:
```
Comment (rarkins) [inline docs/development/configuration.md:5]:
```suggestion
Renovate's bot admin settings can be configured via any of these methods:
```

File: docs/development/configuration.md (hunk 1 above)
Comment (rarkins) [inline docs/development/configuration.md:8]:
```suggestion
- Environment variables
```

File: docs/development/configuration.md [hunk 3]
```diff
@@ -2,10 +2,10 @@
...
 
 - Configuration file
-- Environment
+- Environment (variable? file? what is this about?)
 - CLI
```
Comment (rarkins) [inline docs/development/configuration.md:9]:
```suggestion
- CLI parameters
```

File: docs/development/configuration.md [hunk 4]
```diff
@@ -22,16 +22,16 @@ The above are listed in **_reverse order_** of preference. e.g. CLI values will
 
 ### Default Configuration
 
-Default configuration values can be found in
-[lib/config/definitions.ts](../../lib/config/definitions.ts)
+The default configuration values can be found in [lib/config/definitions.ts](../../lib/config/definitions.ts).
```
Comment (rarkins) [inline docs/development/configuration.md:25]:
```suggestion
The default configuration values can be found in [lib/config/definitions.ts](../../lib/config/definitions.ts).
Options which have `"admin": true` are reserved only for bot admin configuration and cannot be configured within repository config files.
```

**Review by rarkins — CHANGES_REQUESTED** [review]

**Pull request merged**